Resources/Prototypes/_Carpmosia/Loadouts/Misc/Metaplush/screams.yml

```
@@ -0,0 +1,12 @@
+- type: loadout
+  id: LizardPlushieScreams
+  effects:
+  - !type:JobRequirementLoadoutEffect
+    requirement:
+      !type:RoleTimeRequirement
+      role: JobAtmosphericTechnician
+      time: 50h
+  storage:
+    back:
+    - PlushieLizardScreams
+  groupBy: "metaplush"
```

```suggestion
  groupBy: "metaplush"

```
end files with a newline